### PR TITLE
some typo in method binds fixed

### PR DESCRIPTION
**Pull request by ThakeeNathees**

### CODE REVIEW

File: editor/editor_plugin.cpp [hunk 1]
```diff
@@ -293,7 +293,7 @@ void EditorInterface::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_open_scenes"), &EditorInterface::get_open_scenes);
 	ClassDB::bind_method(D_METHOD("get_edited_scene_root"), &EditorInterface::get_edited_scene_root);
 	ClassDB::bind_method(D_METHOD("get_resource_previewer"), &EditorInterface::get_resource_previewer);
-	ClassDB::bind_method(D_METHOD("get_resource_filesystem"), &EditorInterface::get_resource_file_system);
+	ClassDB::bind_method(D_METHOD("get_resource_file_system"), &EditorInterface::get_resource_file_system);
```
Comment (akien-mga) [inline editor/editor_plugin.cpp:296]: That was not a typo, we use "filesystem" in one word. If anything, the C++ method could be renamed, not the public API.

Comment (ThakeeNathees) [reply editor/editor_plugin.cpp:296]: @akien-mga  but in the next 6th line its file_system, is it okey to leave it that way?
https://github.com/godotengine/godot/blob/f7b8d0c688dc15b8841ac13990569dda81bf0652/editor/editor_plugin.cpp#L302

Comment (ThakeeNathees) [reply editor/editor_plugin.cpp:296]: Never mind, changing public API impact users with regressions.

Comment (akien-mga) [reply editor/editor_plugin.cpp]: I guess `get_file_system_dock` could be renamed to `get_filesystem_dock`.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**